policy/policy.example.agent.json

```diff
@@ -0,0 +1,35 @@
+{
+    "description": "hello there, security policies are fun!",
+    "rules": [
+      {
+        "is_stateful": true,
+        "ports": [
+          80,
+          443
+        ],
+        "protocol": "TCP"
+      }
+    ],
+    "peers": [
+      {
+        "segment_network_id": 2
+      }
+    ],
+    "direction": "ingress",
+    "applied_to": [
+      {
+        "segment_network_id": 1,
+        "tenant_network_id": 1
+      }
+    ],
+    "datacenter" : {
```

Comment:
Yeah, I wasn't talking about config files, but about getting the config via the root service. I agree that we don't have to implement this right here, since we have this in Go already. So for now, yeah, I guess we can leave it as is and just need to make sure that the topology service passes this in.